make/tools.mk

```diff
@@ -109,6 +109,10 @@ ifdef WINDOWS
 	$(V1) ./downloads/qt-opensource-windows-x86-mingw492-5.5.0.exe
 endif
 
+	# Configure Qt Creator user project files (GCS and flight firmware)
```

Comment:
How about setting it up as a separate script that is called as part of the SDK install process?